README.md

```diff
@@ -1,3 +1,14 @@
+<a href="https://hub.docker.com/r/codercom/enterprise-base">![Docker Pulls](https://img.shields.io/docker/pulls/codercom/enterprise-base?label=codercom%2Fenterprise-base)</a>
```

Comment:
I'm just a random drive-by commenter that came across this PR by chance, so take this for what it's worth (nothing at all), but is there a reason why you're mixing Markdown and HTML to set the link here?

An approach I've seen in other projects (for example, see [kubevirt's README.md](https://github.com/kubevirt/kubevirt/blob/master/README.md) or [runc's README.md](https://github.com/opencontainers/runc/blob/master/README.md)) is to wrap the image with a Markdown link, for example:

```markdown
[![Docker Pulls](https://img.shields.io/docker/pulls/codercom/enterprise-base?label=codercom%2Fenterprise-base)](https://hub.docker.com/r/codercom/enterprise-base)
```

I think this renders the same way (i.e. the generated HTML is the same), like this:

[![Docker Pulls](https://img.shields.io/docker/pulls/codercom/enterprise-base?label=codercom%2Fenterprise-base)](https://hub.docker.com/r/codercom/enterprise-base)

This is arguably more readable than what the PR currently has (I guess that's a personal preference between Markdown and HTML). I hope this is taken as it was intended: a friendly suggestion from a detail-oriented person that noticed this by chance, trying to be helpful 😄

Reply:
Hey @jawnsy thanks for the catch - TBH, I'm not sure why I did that, I normally wouldn't reach for HTML unless needed.

It should be changed with your suggestion

:ok_hand: